src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.NetStandard10.cs

```diff
@@ -0,0 +1,401 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This (along with TypeDescriptor.NetStandard10.cs) just brings back the original implementation which is extremely stripped down. Trying to if/def the new implementation to get the 4.0.0 surface area for .NET Standard 1.0 proved to be much harder and this seemed like an easier route.